doc-Managing_Infrastructure_and_Inventory/topics/Instance_Evacuation.adoc

```diff
@@ -1,4 +1,4 @@
-If a Compute node is shut down, you can _evacuate_ instances hosted on it. This is only useful if the instances use shared storage or Block Storage volumes. See https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/single/instances-and-images-guide/#section-migration-evacuation[Evacuate Instances] (in the Red Hat OpenStack Platform _Instances and Images Guide_) for details on the underlying OpenStack process.
+If a Compute node is shut down, you can _evacuate_ instances hosted on it. This is only useful if the instances use shared storage or Block Storage volumes. See https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/single/instances-and-images-guide/#section-migration-evacuation[Evacuate Instances] (from the Red Hat OpenStack Platform _Instances and Images Guide_) for details on the underlying OpenStack process.
```

Comment:
Hey Suyog,  
As we've been discussing conventions a fair bit recently, I think we should all stick to the same style with "from"/"in"as well. Our conventions doc here says "in" is correct -- do you still prefer to use "from", do you want to change it back to "in", or should we hold on this merge, and discuss this in a future docs meeting? I also ask because I think I eliminated some "froms" in the links I was updating, and I'd like to agree on one convention :) What do you think?